vscode/microsoft-kiota/src/extension.ts

```diff
@@ -188,6 +190,13 @@ export async function activate(
             await displayGenerationResults(context, openApiTreeProvider, config);
           }
         }
+
+        function getPluginName(): string | undefined {
+          if (openApiTreeProvider.apiTitle) {
+            return openApiTreeProvider.apiTitle.replace(/[^a-zA-Z0-9_]+/g, '');
+          }
+          return undefined;
```

Comment:
Would it be a good idea to have a efault value instead of undefined as a plugin name. do we foresee a scenario where apiTitle does not exist

Reply:
Setting this as undefined satisfies two things. 
1. the apiTitle in the openApiTreeProvider could be undefined.
2. the config that consumes this function if nothing is passed is set as undefined 

When it's undefined, the experience remains as it is before this change is merged

Reply:
OpenAPI title is required per their docs: https://spec.openapis.org/oas/v3.1.0#fixed-fields-0